tests/unit/Endpoints/Webhooks/InfoHandlerTest.php

```diff
@@ -35,6 +35,27 @@ public function testHandleOk()
 
         $this->mockApiCallsFromDirectory(self::DATA_DUMP_FOLDER, false);
 
+        $file = $this->getHookDataDump('hook.info.json');
+        $rest = $this->getRestWithToken($file);
+        $this->assertEquals('info', $file->getHookAction());
+
+        $this->assertPreConfiguration($rest);
+        $this->assertPostConfiguration($rest);
+    }
+
+    private function assertPreConfiguration(\WP_REST_Request $rest): void
+    {
+        $response = $this->handleRequest($rest);
+        $data = $response->get_data();
+
+        $extra = $data['extra'];
+        $activeCapabilities = $extra['active_capability'];
+        $this->assertNotContains('b2s_shipping_method', $activeCapabilities, 'Shipping method should not be a default capability');
+    }
+
+    private function assertPostConfiguration(\WP_REST_Request $rest): void
+    {
+        StoreKeeperOptions::set(StoreKeeperOptions::SHIPPING_METHOD_USED, 'yes');
```

Comment:
```suggestion
```

assert should not "do" stuff, just assert. Move it to testHandleOk

Reply:
Moved.